config-templates/README.md

```diff
@@ -1,16 +1,13 @@
-#### Physics and Computer Science
+#### Computer Science
 
-If you want to get both physics and computer science content working, you'll need two copies of the `segue-config.properties` file. For consistency, use `segue-config.phy.properties` and `segue-config.cs.properties`.
+If you want to get computer science content working, you'll need a copy of the `segue-config.properties` file.
 
 For the Computer Science version, you'll need to update:
 
  - The names and email addresses, if you want to use Computer Science ones.
- - `CONTENT_INDICES_LOCATION` to use `content_indices.cs.properties` (which is needed for the commit SHAs to be valid).
+ - `CONTENT_INDICES_LOCATION` to use `content_indices.properties` (which is needed for the commit SHAs to be valid).
```

Comment:
Is this right? I can still see a `content_indices.cs.properties` and a `content_indices.phy.properties` in the `config-templates` folder.